{"identifier":{"url":"doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent\/ErrorCode\/unexpectedError","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"symbolKind":"case","title":"FinancialConnectionsEvent.ErrorCode.unexpectedError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"unexpectedError","kind":"identifier"}],"roleHeading":"Case","role":"symbol","externalID":"s:10StripeCore25FinancialConnectionsEventV9ErrorCodeO010unexpectedF0yA2EmF","modules":[{"name":"StripeCardScan"}]},"hierarchy":{"paths":[["doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan","doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent","doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent\/ErrorCode"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/stripecardscan\/financialconnectionsevent\/errorcode\/unexpectederror"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"type":"text","text":"An unexpected error occurred, either in an API call or on the client-side."}],"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unexpectedError","kind":"identifier"}],"platforms":["iOS"]}]}],"references":{"doc://com.stripe.stripe-card-scan/documentation/StripeCardScan/FinancialConnectionsEvent/ErrorCode":{"type":"topic","url":"\/documentation\/stripecardscan\/financialconnectionsevent\/errorcode","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"ErrorCode"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ErrorCode"}],"abstract":[],"identifier":"doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent\/ErrorCode","title":"FinancialConnectionsEvent.ErrorCode","role":"symbol"},"doc://com.stripe.stripe-card-scan/documentation/StripeCardScan/FinancialConnectionsEvent":{"type":"topic","url":"\/documentation\/stripecardscan\/financialconnectionsevent","navigatorTitle":[{"kind":"identifier","text":"FinancialConnectionsEvent"}],"title":"FinancialConnectionsEvent","kind":"symbol","abstract":[],"role":"symbol","identifier":"doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"FinancialConnectionsEvent","kind":"identifier"}]},"doc://com.stripe.stripe-card-scan/documentation/StripeCardScan":{"type":"topic","url":"\/documentation\/stripecardscan","kind":"symbol","abstract":[{"type":"text","text":"This library provides support for the standalone Stripe CardScan product."}],"role":"collection","title":"StripeCardScan","identifier":"doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan"},"doc://com.stripe.stripe-card-scan/documentation/StripeCardScan/FinancialConnectionsEvent/ErrorCode/unexpectedError":{"type":"topic","identifier":"doc:\/\/com.stripe.stripe-card-scan\/documentation\/StripeCardScan\/FinancialConnectionsEvent\/ErrorCode\/unexpectedError","kind":"symbol","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unexpectedError"}],"url":"\/documentation\/stripecardscan\/financialconnectionsevent\/errorcode\/unexpectederror","abstract":[{"type":"text","text":"An unexpected error occurred, either in an API call or on the client-side."}],"title":"FinancialConnectionsEvent.ErrorCode.unexpectedError"}}}